### DISCUSSION TITLE
Run Firefly III as a user different from root

### DISCUSSION THREAD

**Discussion by steakhutzeee** [firefly-iii/firefly-iii#4852] [General]
I am running Firefly III version 5.5.12

**Description**
Hi, i'm using Firefly III for quite some time now and noted that the container is running as root. Can i run it instead as another user?

**Extra info**
```
Running as 'root' in group 'root'.
Current working dir is '/var/www/html'
```

The database is linuxserver/mariadb running with the same user i would like to run Firefly III.

Thanks in advance!

**Comment by JC5** [comment]
I haven't found a way to do so reliably, if you have pointers let me know.

**Reply by steakhutzeee** [reply]
> 
> 
> I haven't found a way to do so reliably, if you have pointers let me know.

With linuxserver containers i usually use PGID and GUID env variables.

With others i used the tag _user:_ in the docker-compose.

So you found issue running as a user differnt from root?

**Reply by JC5** [reply]
The start up scripts and build scripts need to be ready for different users as well.
